New `<meta name=application-title>` meta name available in Chromium (Edge/Chrome 134) (#37852)
mdn/content: files/en-us/web/html/element/meta/name/index.md

  > - Some values are mutually exclusive, like `index` and `noindex`, or `follow` and `nofollow`. In these cases the robot's behavior is undefined and may vary between them.
  > - Some crawler robots, like Google, Yahoo and Bing, support the same values for the HTTP header {{HTTPHeader("X-Robots-Tag")}}; this allows non-HTML documents like images to use these rules.

#### Other names

> - `application-title`: used by web applications that are installed as standalone apps on supporting desktop operating systems to customize an app's title bar. Although the text content of the {{HTMLElement("title")}} element is usually displayed in browser tabs when the app is running in a browser, the `application-title` metadata name can be used to set a different title for the app when it is running as a standalone installed app.

Reviewer: ```suggestion
- `application-title`: Used to customize an app's title bar for web applications installed as standalone apps on supporting desktop operating systems. While the text content of the {{HTMLElement("title")}} element is usually displayed in browser tabs when the app is running in a browser, the `application-title` metadata name can be used to set a different title for the application when it is running as a standalone installed app.
```
